packages/aws-cdk-lib/aws-ecs/README.md

```diff
@@ -156,6 +156,19 @@ const autoScalingGroup = new autoscaling.AutoScalingGroup(this, 'ASG', {
 });
 ```
 
+To customize the cache key use the `additionalCacheKey` parameter.
```

Comment:
You need to explain WHY a user might want to do this. If I didn't know what this was already, reading this bit of the documentation certainly wouldn't help me.